samples/WebFormsSample/WebFormsSample.Dynamic.csproj

```diff
@@ -15,4 +15,10 @@
     </Content>
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="EntityFramework" Version="6.5.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
```

Comment:
is newtonsoft required?

Reply:
no.. came as a part of testing.. removed